src/popup/bookmarkModule.js

```diff
@@ -207,6 +225,38 @@ function loadImages() {
 
           removeSpinner(elements.spinnerPlaceholderBookmarks);
           appendToGrid(msnry, fragment, gridItemDiv, elements.gridBookmarks);
+        })
+        .then(() => {
+          const btn = elements.selectButtons[elements.selectButtons.length - 1];
+
+          btn.isChecked = false;
+          bookmarkDOM[btn.getAttribute('id')] = btn;
+
+          btn.addEventListener('click', () => {
+            if (btn.isChecked) {
+              btn.parentElement.removeAttribute('style');
+              selectedBookmarks -= 1;
+            } else {
+              btn.parentElement.setAttribute('style', 'opacity : 1');
+              selectedBookmarks += 1;
+            }
+            btn.isChecked = !btn.isChecked;
+
+            console.log(selectedBookmarks);
+            if (selectedBookmarks === elements.selectButtons.length) {
+              elements.selectAllButton.innerHTML = `
```

Comment:
Please do not use `innerHTML`([reason](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#Security_considerations)). Instead use `textContent` to change the text directly inside the `.content` div.

Reply:
Thanks for clarifying link.